lib/fizzy/parser.cpp

```diff
@@ -152,7 +152,8 @@ parser_result<std::string> parse_string(const uint8_t* pos, const uint8_t* end)
     if (!utf8_validate(pos, pos + size))
         throw parser_error{"Invalid UTF-8"};
 
-    auto ret = std::string(pos, pos + size);
+    const auto char_begin = reinterpret_cast<const char*>(pos);
```

Comment:
Shouldn't this go into #204 ?

Reply:
It is cherry picked from there. Needed to have it running.